pyspecProcScripts/phasing.py

```diff
@@ -186,12 +186,44 @@ def costfun(ph1):
     # }}}
 
 
-def fid_from_echo(d, signal_pathway, fl=None, add_rising=False, fraction_nonnoise=0.1, direct="t2",
+def fid_from_echo(d, signal_pathway, fl=None, add_rising=False, direct="t2",
         exclude_rising=3,
         slice_multiplier=20,
         peak_lower_thresh=0.1,
         show_hermitian_sign_flipped=False,
         show_shifted_residuals=False):
+    """
+    Parameters
+    ==========
+    signal_pathway: dict
+                    coherence transfer pathways that correspond to the signal
+    fl:             figlist or None (default)
+                    If you want the diagnostic plots (showing the
+                    distribution of the data in the complex plane),
+                    set this to your figlist object.
+    add_rising:     boolean
+                    when true the script will add a rising edge to the echo
```

Comment:
It's taking the first part of the echo... The part that rises up to the maximum, and it's adding it to the decaying (FID like) part. We want to do this if it works well in order to increase the SNR of the early points of the signal... But of course, there are some potential pitfalls